cli/cfncluster/cfncluster.py

```diff
@@ -488,16 +444,13 @@ def delete(args):
             sys.stdout.write('\n')
             sys.stdout.flush()
         if status == 'DELETE_FAILED':
-            logger.info('Cluster did not delete successfully. Run \'cluster delete %s\' again' % stack)
-    except boto.exception.BotoServerError as e:
-        if e.message.endswith("does not exist"):
-            #sys.stdout.write('\r\n')
-            if not saw_update:
```

Comment:
I've updated this to print `cluster deleted successfully` when the `saw_update` flag is set to true. Otherwise it does the normal ClientError logic where it prints the error message which results in the `cluster x does not exist` message.